<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_video">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>720</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_video_file_path">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>470</y>
      <width>281</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_descriptor">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>500</y>
      <width>281</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>470</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Video</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Fern_mat</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_camera">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>470</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Camera</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_select_path_video">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>470</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Select video</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_fern">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>490</y>
      <width>96</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Fern</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_orb">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>510</y>
      <width>96</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>ORB</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>540</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Current frame:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_frame_id">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>540</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>      --//--</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_clust">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>550</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Clusterization 
for ORB</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_track_orb">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>530</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Tracking by ORB</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Hungarian">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>590</y>
      <width>96</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Hungarian</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_max_frames_to_skip">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>580</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nkeypoints">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>580</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_dist_tresh">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>580</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>570</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Max count of &lt;br/&gt;frames to skip&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>580</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nkeypoints</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>580</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Distance threshold</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_proc_time">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>540</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>540</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Processing time</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_fps">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>540</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>--//--</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>540</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>FPS</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_orb_area">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>620</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>620</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ORB area</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_hung_orb">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>620</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Hungarian - ORB</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>620</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>BRIEF size</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_brief_size">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>620</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_video">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>430</y>
      <width>661</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_cur_frame">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>430</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_harris_corn">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>640</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Good Features to track</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_eco">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>670</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ECO</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>909</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
